### PR TITLE
[FIX] open_academy: pre-commit suggested lints T#52898

### PR DESCRIPTION
**Pull request by rolandojduartem**
Now, the pylint tests are passed

Screenshot from 2022-01-17 16-16-22

### CODE REVIEW

**Comment by rolandojduartem** [comment]
@frahikLV , test are passing, could you review this one?

File: open_academy/models/session.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from odoo import models, fields, api
 from odoo.exceptions import ValidationError
+from odoo.tools.translate import _
```
Comment (frahikLV) [inline open_academy/models/session.py:3]: remove this line, odoo has the translation module
```suggestion
from odoo import models, fields, api, _
```

**Review by frahikLV — CHANGES_REQUESTED** [review]
@rolandojduartem Can you address these changes?

Also, on README.rst file, please add a descriptive information, something like this example:

```rst
Open academy
============
This is a training module; it has been created following the guidelines established in https://www.odoo.com/documentation/15.0/developer/howtos/backend.html
Author: name <vauxoo_email>
```

**Comment by rolandojduartem** [comment]
@frahikLV , I applied the changes that you suggested, could you review it, again?

**Review by frahikLV — APPROVED** [review]
@rolandojduartem Great! I will merge it

**Pull request merged**